.changeset/lucky-zebras-serve.md

```diff
@@ -0,0 +1,9 @@
+---
+"@kaizen/components": minor
+---
+
+- remove dep on @kaizen/draft-modal
```

Comment:
```suggestion
- remove dep on @kaizen/draft-modal as the styles causes compile to fail
```

Something like that so it doesn't sound like we're just removing it to go backwards :P

Reply:
I hummed and had on that one, I'm never really sure how much context to put in a changeset vs what in the PR description. Happy to do so but feel like we need that doc on how best to write a changeset, least for our team

Reply:
Yep, that definitely came up in WoW as something we need to figure out. I guess I'd say it should list the change made and how it solves the problem? It's a fine line though, so we definitely need some sort of convention 😬